index.html

```diff
@@ -76,6 +76,40 @@ <h2>A project for first-time contributions</h2>
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
+        
+            <!-- ________ Wangari's card START ________  -->
+  <div class="card">
+    <p class="name">Susan Wangari</p>
+    <p class="contact">
+      <i class="fab fa-github"></i>
+      <a href="https://github.com/Susan-Wangari" target="_blank">Susan-Wangari</a>
+    </p>
+    <p class="about">
+     Iam a Software Developer working with JavaScript and Currently learning NodeJS. When free I love doing crochet. I am also a student.<br>
+      <a href="https://twitter.com/coder_muchacha" target="_blank">Twitter: @coder_muchacha</a>.
```

Comment:
Hello @Susan-Wangari, you will want to move your contact information up under the Contact paragraph. This will allow the stylings we have in place for the card to take affect on this element.